crm_location_nuts/models/__init__.py

```diff
@@ -1,3 +1,2 @@
-# -*- coding: utf-8 -*-
 
```

Comment:
You should remove the blank lines remaining at the beginning of this file (and some others) as well (or replace them with a license line - my personal preference).